packages/vscode-extension/src/debug/teamsAppInstallation.ts

```diff
@@ -17,27 +17,17 @@ import { openUrlWithNewProfile } from "./launch";
 
 export async function showInstallAppInTeamsMessage(env: string, appId: string): Promise<boolean> {
   const isLocal = env === environmentManager.getLocalEnvName();
-  const botId = await commonUtils.getBotId(env);
+  const botId = isV3Enabled() ? undefined : await commonUtils.getBotId(env);
 
   let messages: string[] = [];
   const items = [localize("teamstoolkit.localDebug.installApp.installInTeams")];
   if (isV3Enabled()) {
     if (isLocal) {
-      if (botId) {
-        messages = [
-          localize("teamstoolkit.localDebug.installApp.bot.description"),
```

Comment:
Could these strings be removed from package.nls.json?

Reply:
They are still used for v2 projects. We can remove them at least after v3 is public and migration is done.